src/App.jsx

```diff
@@ -14,6 +15,7 @@ function App() {
         <Routes>
           <Route exact path="/" element={<Home />} />
           <Route exact path="/reviews" element={<ReviewList />} />
+          <Route path = "/reviews/:reviewId" element={<ReviewCard />} />
```

Comment:
How are users getting here? You have no navigation to this element at all right now.

Reply:
I cannot navigate to this element from anywhere in this app.